components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/reporting-overview.component.ts

```diff
@@ -114,6 +115,20 @@ export class ReportingOverviewComponent implements OnInit, OnDestroy {
     }
   }
 
+  onDateChanged(endDate) {
+    const queryParams = {...this.route.snapshot.queryParams};
+    const endDateMoment = moment(endDate);
+    if (endDate && endDateMoment.isValid()) {
+      if (moment().utc().format('YYYY-MM-DD') === endDateMoment.format('YYYY-MM-DD')) {
```

Comment:
i'm trying to pull down the change to test it - but my internet is kinda shitty on the plane.
i'm super curious about this logic tho -- if the end date that was selected is the same as today utc, we delete the end time filter from the query params? why? or am i reading this incorrectly?

Reply:
@vjeffrey Since today's date is the default filter selection, if we remove the date filter from the URL today's date will be selected. 
Trying to prevent having the default filter selections in the URL. 

Thank you for taking a look. I am excited to get this merged. I also have some other clickable items coming next!

Reply:
👍 makes sense; thanks for explaining!

Reply:
Is there a way to add a test for this case?

Reply:
Thanks! added.